services/core-web/src/components/mine/Projects/ProjectSummary.tsx

```diff
@@ -125,202 +105,103 @@ export const ProjectSummary: FC = () => {
     return payload;
   };
 
-  const transformAuthorizations = (valuesFromForm: any) => {
-    const { authorizations = {}, project_summary_guid } = valuesFromForm;
-
-    const transformAuthorization = (type, authorization) => {
-      return { ...authorization, project_summary_authorization_type: type, project_summary_guid };
-    };
-
-    let updatedAuthorizations = [];
-    let newAmsAuthorizations = [];
-    let amendAmsAuthorizations = [];
-
-    projectSummaryAuthorizationTypesArray.forEach((type) => {
-      const authsOfType = authorizations[type];
-      if (authsOfType) {
-        if (isArray(authsOfType)) {
-          const formattedAuthorizations = authsOfType.map((a) => {
-            return transformAuthorization(type, a);
-          });
-          updatedAuthorizations = updatedAuthorizations.concat(formattedAuthorizations);
-        } else {
-          newAmsAuthorizations = newAmsAuthorizations.concat(
-            authsOfType?.NEW.map((a) =>
-              transformAuthorization(type, { ...a, project_summary_permit_type: ["NEW"] })
-            )
-          );
-          amendAmsAuthorizations = amendAmsAuthorizations.concat(
-            authsOfType?.AMENDMENT.map((a) =>
-              transformAuthorization(type, { ...a, project_summary_permit_type: ["AMENDMENT"] })
-            )
-          );
-        }
-      }
+  const handleCreateProjectSummary = async (values, message) => {
+    return dispatch(
+      createProjectSummary(
+        {
+          mineGuid,
+        },
+        values,
+        message
+      )
+    ).then(({ data: { project_guid, project_summary_guid } }) => {
+      history.replace(
+        routes.EDIT_PROJECT_SUMMARY.dynamicRoute(
+          project_guid,
+          project_summary_guid,
+          projectFormTabs[1],
+          false
+        )
+      );
     });
-    return {
-      authorizations: updatedAuthorizations,
-      ams_authorizations: { amendments: amendAmsAuthorizations, new: newAmsAuthorizations },
-    };
   };
 
-  const handleTransformPayload = (payload) => {
-    const values = removeUploadedDocument(payload, uploadedDocuments);
-    let payloadValues: any = {};
-    const updatedAuthorizations = transformAuthorizations(values);
-    payloadValues = {
-      ...values,
-      ...updatedAuthorizations,
-    };
-
-    delete payloadValues.authorizationTypes;
-    return payloadValues;
-  };
-
-  const handleUpdate = (message) => {
-    const { project_guid, mrc_review_required, contacts, project_lead_party_guid } = formValues;
+  const handleUpdateProjectSummary = async (payload, message) => {
```

Comment:
The combo of .then and async await here feels a bit clunky.  Is it possible to make it all async/await?

Reply:
That's probably a good thing to do. I don't want to expand the scope _too_ much though, and want to unblock AMS work so I'll leave it as something to keep in mind for the future.